### ISSUE TITLE
EDDP Watches not working

### ISSUE THREAD

**Issue by Smash-Time** [cmdrmcdonald/EliteDangerousDataProvider#167]
I'm using the EDDI plugin with HCS voicepacks. The watches in EDDP monitor worked prior to me accidentally deleting my Oculus folder which deleted my ED folder as well. After I reinstalled ED and reset my Netlog monitor path in EDDI, I'm not able to get any watches to work in EDDP. I've setup multiple watches with distance from ship set at 60 and distance from home (Aulin) set at 120 as well as a "anything" watch with no distances set but I'm not get any voice reports back. Is there anything I can do to trouble shoot the problem?

**Comment by cmdrmcdonald** [comment]
Can you confirm that the plugin is enabled?  There will be a checkbox for it on the 'EDDP monitor' tab of the EDDI UI.

If so, please could you attach your %APPDATA%\EDDI\eddpmonitor.json file so that I can take a look?

**Comment by Smash-Time** [comment]
Yes, I do have it enabled. I moved to a different system (Ceos) to get allied with Sirius and now the watch reports are coming in. I thought for sure that I screwed it up by deleting my Oculus folder but it appears that there just wasn't anything to watch around the Aulin area (???). I guess stranger things have happened. BTW, thanks so much for creating this program! It's a huge help. Especially when using the Rift. I had heard that HCS was not going to include EDDI in upcoming versions of their VA profiles. I hope that's not the case. If it is, I hope I can still use the EDDI plugin without any conflicts. Thanks again!
